ci_tendrl_cluster.yml

```diff
@@ -0,0 +1,58 @@
+---
+# Perform following tasks, when applicable
+# (e.g. there are nodes in gluster or ceph_* group)
+# * Configure firewall
+# * Install Tendrl server.
+# * Install Tendrl storage nodes.
+# * Add Gluster repo.
+# * Add Ceph repos.
+- include: qe_pre_installation_tasks.yml
+
+# configure firewall
+- include: firewall_tendrl.yml
+- include: firewall_tendrl_workaround.yml
+
+- include: qe_reboot.yml
+
+- include: tendrl_server.yml
+
+- include: tendrl_node.yml
+
+# Add Centos Gluster YUM repo
+- hosts: gluster
+  remote_user: root
+  roles:
+    - { role: gluster-centos-repo, when: groups.gluster is defined }
```

Comment:
Is this condition `when: groups.gluster is defined` needed considering that you apply this role on members of this group in the first place?